app/jenkins-for-jira-ui/src/components/InProductHelpDrawer/InProductHelpAction.tsx

```diff
@@ -20,24 +31,60 @@ export enum InProductHelpActionType {
 type InProductHelpActionProps = {
 	label: string,
 	type: InProductHelpActionType,
-	appearance?: InProductHelpActionButtonAppearance
+	appearance?: InProductHelpActionButtonAppearance,
+	screenName?: string
+};
+
+const analyticsClient = new AnalyticsClient();
+
+const iphClickSource = (screenName?: string): string => {
+	switch (screenName) {
+		case SET_UP_GUIDE_SCREEN_NAME:
+			return AnalyticsScreenEventsEnum.ServerManagementScreenName;
+			break;
+		case CONNECTION_WIZARD_SCREEN_NAME:
+			return AnalyticsScreenEventsEnum.ConnectionWizardScreenName;
+			break;
+		case JENKINS_SETUP_SCREEN_NAME:
+			return AnalyticsScreenEventsEnum.JenkinsSetupScreenName;
+			break;
+		default:
+			return '';
+			break;
+	}
 };
 
 export const InProductHelpAction = ({
 	label,
 	type,
-	appearance
+	appearance,
+	screenName
 }: InProductHelpActionProps): JSX.Element => {
 	const [isDrawerOpen, setIsDrawerOpen] = useState(false);
 	const inProductHelpTypeClassName =
-		type === InProductHelpActionType.HelpLink ? inProductHelpActionLink : inProductHelpActionButton;
+		type === InProductHelpActionType.HelpLink
+			? inProductHelpActionLink
+			: inProductHelpActionButton;
 	const actionRole = InProductHelpActionType.HelpLink ? 'link' : 'button';
 	const inProductHelpButtonStyles =
 		appearance === InProductHelpActionButtonAppearance.Primary
-			? inProductHelpActionButtonPrimary : inProductHelpActionButtonDefault;
+			? inProductHelpActionButtonPrimary
+			: inProductHelpActionButtonDefault;
+
+	const actionSubject = type === InProductHelpActionType.HelpButton ? 'button' : 'link';
 
-	const openDrawer = () => {
+	const openDrawer = async () => {
 		setIsDrawerOpen(true);
+
+		await analyticsClient.sendAnalytics(
+			AnalyticsEventTypes.UiEvent,
+			AnalyticsUiEventsEnum.OpenInProductionHelpDrawerName,
+			{
+				source: iphClickSource(screenName) || '',
```

Comment:
/optional - do we want any error handling here? its after setIsDrawerOpen(true); so hopefully it fails quietly anyway

Reply:
I don't think it's needed. Even if the analytics request fails the drawer would already be opened.